ykpack/src/types.rs

```diff
@@ -175,6 +175,9 @@ pub enum Statement {
     /// Information about which locals are currently live/dead.
     StorageLive(Local),
     StorageDead(Local),
+    /// A (non-inlined) call from a TIR trace to a binary symbol using the system ABI. This does
+    /// not appear in SIR.
```

Comment:
`in the SIR`.

Reply:
I  think `in SIR` is more correct. We are saying that this kind of statement cannot ever appear in any instance of SIR.